ContentBlocks/ContentElements/carousel/templates/frontend.html

```diff
@@ -0,0 +1,31 @@
+<f:asset.css identifier="content-block-css-t3docs-t3docs/carousel" href="{cb:assetPath()}/frontend.css"/>
+<f:asset.script identifier="content-block-js-t3docs-t3docs/carousel" src="{cb:assetPath()}/frontend.js"/>
+
+<div id="carousel{data.uid}" class="carousel slide" data-bs-ride="carousel">
+    <div class="carousel-indicators">
+        <f:for each="{data.t3docs_carousel_carousel_items}" as="item" iteration="iteration">
```

Comment:
You can omit the prefix `t3docs_carousel_`

Reply:
Thanks, good hint

Reply:
Done